src/content/documentation/publish/distribute-manifest-versions.md

```diff
@@ -49,7 +49,26 @@ If you choose to remain on MV2 and wait to transition to MV3 later, there are st
 
 {% capture content %}
 
-## How to sign an MV3 version
+## Prepare your MV3 version
+
+Because you are distributing your extension from AMO and a third-party website, you need to specify the origins of the sites where the MV3 version's XPI file is hosted in the [`install_origins`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/install_origins) manifest key.
```

Comment:
"Because you are distributing your extension from AMO and a third-party website," -> "If you are distributing your extension from third-party website (not AMO),"?

Reply:
Also please use locale-less URLs for MDN (i.e. remove `/en-US`)

Reply:
@willdurand URLs are now locale-less

Reply:
well, that is the point of the article, yes.  But you need to specify `install_origins` specifically if you are distributing the MV3 version from a 3rd party website (we've just said "your website" elsewhere, I believe).  You don't have to distribute it via a website.

Reply:
I'm not sure the best way to get that across.

Reply:
@eviljeff I see your point, I have reworded; please take a look.